vscode-dotnet-runtime-library/src/Utils/WebRequestWorker.ts

```diff
@@ -225,7 +225,7 @@ export class WebRequestWorker
         }
         catch(error : any)
         {
-            if(error?.message?.contains('ENOSPC'))
+            if(error && error?.message && (error?.message as string)?.includes('ENOSPC'))
```

Comment:
It does, that new check is unnecessary, but I don't want to change it for this release since we have a build. I will change it after this going forward :)

I thought maybe it was not behaving correctly because it was calling .contains on the object but the problem was .contains does not exist on a string type, not that it was trying .contains on a null type